ui/pages/swaps/index.js

```diff
@@ -111,8 +112,15 @@ export default function Swap() {
   const chainId = useSelector(getCurrentChainId);
   const isSwapsChain = useSelector(getIsSwapsChain);
   const useNewSwapsApi = useSelector(getUseNewSwapsApi);
+  const EIP1559Network = useSelector(isEIP1559Network);
   const fromToken = useSelector(getFromToken);
 
+  if (EIP1559Network) {
```

Comment:
Do we have to have this `if` block and eslint disable? Will everything still work if it is removed?

Reply:
Originally I didn't have this `if` block, but then I realized we don't need to make unnecessary API calls for non EIP-1559 networks in Swaps via this hook. For non EIP-1559 networks we get gas prices like this: https://github.com/MetaMask/metamask-extension/pull/11635/files#diff-f89a0460c9cb04858f4de957f65d6aab869c1d766f11781b1fe8a61f3321620eR199